Config.psm1

```diff
@@ -54,10 +54,11 @@ function Get-AvailableConfigOptions {
         @{"Name" = "install_maas_hooks"; "DefaultValue" = $false; "AsBoolean" = $true;
           "Description" = "If set to true, MAAS Windows curtin hooks will be copied to the image root directory."},
          @{"Name" = "compression_format";
-          "Description" = "Select between tar, gz, zip formats or any combination between these."},
+          "Description" = "Select between tar, gz, tar.gz or zip formats"},
         @{"Name" = "zip_password";
           "Description" = "If this parameter is set, after the image is generated,
-                           a password protected zip archive with the image will be created."},
+                           a password protected zip archive with the image will be created. Previous parameter, compression_format, 
```

Comment:
line too long, keep the line length to be at least reviewable without horizontal scrolling in github.